dll/win32/shell32/lang/hu-HU.rc

```diff
@@ -378,335 +378,334 @@ END
-    LTEXT "Volume &label", -1, 7, 93, 170, 9
+    LTEXT "Kötet&címke", -1, 7, 93, 170, 9
     EDITTEXT 28679, 7, 103, 170, 13, ES_AUTOHSCROLL
-    GROUPBOX "Format &options", 4610, 7, 121, 170, 49
-    AUTOCHECKBOX "&Quick Format", 28674, 16, 135, 155, 10
-    AUTOCHECKBOX "&Enable Compression", 28675, 16, 152, 155, 10
+    GROUPBOX "Formázási &beállítások", 4610, 7, 121, 170, 49
+    AUTOCHECKBOX "&Gyorsformázás", 28674, 16, 135, 155, 10
+    AUTOCHECKBOX "&Tömörítés engedélyezése", 28675, 16, 152, 155, 10
 END
 
 IDD_CHECK_DISK DIALOGEX 50, 50, 194, 115
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Check Disk"
+CAPTION "Lemezellenőrzés"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON "Start", IDOK, 63, 95, 60, 14
-    GROUPBOX "Check disk options", -1, 7, 6, 179, 40
+    DEFPUSHBUTTON "Indítás", IDOK, 63, 95, 60, 14
+    GROUPBOX "Lemezellenőrzés beállításai", -1, 7, 6, 179, 40
     PUSHBUTTON "Mégse", IDCANCEL, 128, 95, 60, 14
-    AUTOCHECKBOX "Automatically fix file system errors", 14000, 12, 15, 155, 10
-    AUTOCHECKBOX "&Scan for and attempt recovery of bad sectors", 14001, 16, 30, 165, 10
+    AUTOCHECKBOX "Fájlrendszer hibáinak automatikus javítása", 14000, 16, 17, 155, 10
+    AUTOCHECKBOX "&Szektorhibák keresése és kísérlet a javításukra.", 14001, 16, 30, 163, 10
     CONTROL "", 14002, "MSCTLS_PROGRESS32", 16, 7, 50, 179, 8
     LTEXT "", 14003, 60, 80, 170, 10
 END
 
 IDD_PICK_ICON DIALOGEX 0, 0, 193, 200
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Change Icon"
+CAPTION "Ikoncsere"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    LTEXT "Look for icons in this file:", -1, 7, 7, 179, 10
-    PUSHBUTTON "Browse...", IDC_BUTTON_PATH, 135, 17, 50, 14
+    LTEXT "Ikonok keresése ebben a fájlban:", -1, 7, 7, 179, 10
+    PUSHBUTTON "Tallózás...", IDC_BUTTON_PATH, 135, 17, 50, 14
     EDITTEXT IDC_EDIT_PATH, 7, 17, 123, 14, ES_AUTOHSCROLL
-    LTEXT "Select an icon from the list below:", -1, 7, 36, 179, 10
+    LTEXT "Válasszon egy ikont a listából:", -1, 7, 36, 179, 10
     LISTBOX IDC_PICKICON_LIST, 7, 47, 181, 122, LBS_OWNERDRAWFIXED | LBS_NOINTEGRALHEIGHT | LBS_MULTICOLUMN | LBS_DISABLENOSCROLL | WS_HSCROLL | WS_TABSTOP
     DEFPUSHBUTTON "OK", IDOK, 81, 179, 50, 14
     PUSHBUTTON "Mégse", IDCANCEL, 135, 179, 50, 14
 END
 
 IDD_NOOPEN DIALOGEX 0, 0, 240, 75
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Caution"
+CAPTION "Vigyázat"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_SHELL_SYSTEM_FILE, IDI_SHELL_SYSTEM_FILE, 5, 5, 21, 20
-    LTEXT "This file could either be used by the operating system or by\n\
-another application. Modifying this file might damage your\n\
-system or make it less functional.\n\n\
-Are you sure you want to open this file?", IDC_STATIC, 35, 5, 230, 60
+    LTEXT "Ezt a fájlt az operációs rendszer vagy egy másik alkalmazás használhatja. \
```

Comment:
You don't use the forced newlines `\n` ?

Reply:
I forgot these, thanks!